### PR TITLE
Fix WORKSPACE by updating Bazel Go and Docker rules

### PR DESCRIPTION
**Pull request by jotadrilo**
We are updating some rules and add some missing rules to be able to push the images to GCR.

### CODE REVIEW

**Comment by atomatt** [comment]
The PR's title is misleading, assuming it's not actually broken atm. Maybe, "Update Bazel Go and Docker rules" instead?

File: WORKSPACE [hunk 1]
```diff
@@ -47,6 +47,19 @@ load(
     docker_defaults = "defaults",
 )
 
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_pull",
+    "container_image",
+)
```
Comment (atomatt) [inline WORKSPACE]: Are these needed? IIRC, I think I'd normally expect to see `container_image` in a BUILD file.

**Comment by jotadrilo** [comment]
I usually find this load in the `WORKSPACE` directly. Would you use a different structure?

File: WORKSPACE (hunk 1 above)
Comment (jotadrilo) [reply WORKSPACE]: > The PR's title is misleading, assuming it's not actually broken atm. Maybe, "Update Bazel Go and Docker rules" instead?

yes, it is actually broken :(

Comment (jotadrilo) [reply WORKSPACE]: > Are these needed? IIRC, I think I'd normally expect to see `container_image` in a BUILD file.

Update Bazel Go and Docker rules

**Pull request merged**